src/ChargeDrift/ChargeDrift.jl

```diff
@@ -85,6 +85,14 @@ end
     pt_car in point_types || pt_car in det
 end
 
+function project_to_plane(v⃗::AbstractArray, n⃗::AbstractArray) #Vector to be projected, #normal vector of plane

```

Comment:
This function used to be in the old `src/Geometries` folder and is needed for the old drift code implementation.
However, this will not be needed in the future, when the drift code is refactored.